src/js/containers/covid19/homepage/CovidHighlights.jsx

```diff
@@ -304,13 +307,14 @@ export class CovidHighlights extends React.Component {
             isDateLoading,
             hasNext
         } = this.state;
-        const { totalSpendingAmount, latestSubmissionPeriod } = this.props;
-        const parsedDate = latestSubmissionPeriod
-            ? latestSubmissionPeriod.format('MMMM DD[,] YYYY')
+        const { totalSpendingAmount, submissionPeriods } = this.props;
+        const parsedDate = submissionPeriods.length
+            ? getLatestPeriodAsMoment(submissionPeriods).format('MMMM DD[,] YYYY')
```

Comment:
Should the homepage display January 8, 2021 with these changes? Is it based on `submission_reveal_date`? I see that submission in redux but it is still showing Nov 30.